yal/assessments.py

```diff
@@ -150,3 +154,42 @@ async def state_survey_process_answer(self):
                 self.save_metadata("assessment_score", score)
 
         return await self.go_to_state("state_survey_question")
+
+    async def state_assessment_later_submit(self):
+        msisdn = normalise_phonenumber(self.inbound.from_addr)
+        whatsapp_id = msisdn.lstrip(" + ")
+
+        reminder_time = get_current_datetime() + timedelta(hours=23)
+        self.save_metadata("assessment_reminder", reminder_time.isoformat())
+        assessment_name = self.user.metadata.get(
+            "assessment_name", "sexual_health_literacy"
+        )
+
+        data = {
+            "assessment_reminder": reminder_time.isoformat(),
+            "assessment_name": assessment_name,
```

Comment:
Could we maybe add these to `docs/contact_fields.md` so long, so that we can start keeping track of all the rapidpro contact fields and how they are used.